src/Sylius/Bundle/ApiBundle/Command/Cart/RemoveItemFromCart.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Cart;
+
+final class RemoveItemFromCart
+{
+    /** @var string|null */
+    public $tokenValue;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $productCode;
```

Comment:
We cannot remove items by `productCode`, we need to provide `orderItemId` and based on this choose a proper item to remove